src/cudaautotune/autotuning/tuner.py

```diff
@@ -0,0 +1,149 @@
+import argparse
```

Comment:
How about adding
```suggestion
#!/usr/bin/env python3
import argparse
```
so that the script could be run "directly" (./src/cudaautotune/autotuning/tuner.py)?